contributing/localization.md

```diff
@@ -68,7 +68,7 @@ Here is a more concrete example:
      DEFAULT_LOCALE,
      Localization,
      Message
-   } from '@salesforce/salesforcedx-utils-vscode/out/src/i18n';
+   } from '@salesforce/salesforcedx-utils-vscode';
```

Comment:
I updated places where we were referencing down into the utils package instead of pulling the main module. .